### ISSUE TITLE
menu shortcut for the app

### ISSUE THREAD

**Issue by rodrigoesborges** [sualko/cloud_bbb#52]
@sualko thanks a lot for putting this up. I had known you did a bot at first, didn't you? I hadn't the time to try to use it though.

I wonder why isn't there a shortcut in the top menu bar?

**Comment by sualko** [comment]
> I had known you did a bot at first, didn't you?

What do you mean by bot?

> I wonder why isn't there a shortcut in the top menu bar?

Why should there be a button? You have to configure rooms only once and therefore those settings are at the settings page.

**Comment by rodrigoesborges** [comment]
Sorry for the unclear and bundersome messages . 

Forget about the bot, I just mistook you for another dev that wrote a Nextcloud talk bot , but it was for Jitsi. Sorry about that.


As for the top menu bar, I probably got the wrong view from the way the nextcloud talk app works. In this case, room creation and deletion can happen frequently, for different groups / intentions. Would the metaphor behind the BBB integration app be, then,  to think like physical rooms ?

**Comment by rodrigoesborges** [comment]
But this would be a duplicate of #31 . So closing now and continuing there.
